src/Symfony/Component/CssSelector/README.md

```diff
@@ -36,12 +36,15 @@ names are lower-cased, the following extra pseudo-classes are supported:
 Resources
 ---------
 
+  * [Documentation](https://symfony.com/doc/current/components/css_selector.html)
+  * [Contributing](https://symfony.com/doc/current/contributing/index.html)
+  * [Report issues](https://github.com/symfony/symfony/issues) and
+    [send Pull Requests](https://github.com/symfony/symfony/pulls)
+    in the [main Symfony repository](https://github.com/symfony/symfony)
+
+Third Party Dependencies
```

Comment:
I've changed it by `Credits`, which is the term used by Google to mention the licenses of the projects they use:

![google_chrome_credits](https://cloud.githubusercontent.com/assets/73419/13494350/f22efc1c-e143-11e5-8d7f-e8706eb1def4.png)